dotCMS/dependencies.gradle

```diff
@@ -409,7 +409,7 @@ dependencies {
     compile group: 'com.twelvemonkeys.servlet', name: 'servlet', version:'3.7.0'
     
     
-    compile 'com.github.ben-manes.caffeine:caffeine:2.4.0'
+    compile 'com.github.ben-manes.caffeine:caffeine:2.5.0'
```

Comment:
We follow semver strictly. Please upgrade to the latest 2.x to minimize bugs and have the best experience.

Reply:
Sure thanks @ben-manes